Category/categoryController.js

```diff
@@ -0,0 +1,12 @@
+const Router = require('express').Router()
+const Categorydb = require('./category');
+
+Router.post('/',(req,res) => {
+    const obj = req.body;
+    Categorydb.create(obj,(err,newcategory) => {
+        if (err) res.status(500).json(err);
+        res.status(202).json(newcategory);
```

Comment:
HTTP status code should be `201`, since we're creating a new document and sending it to our client (Postman).

https://httpstatuses.com/201